src/main/kotlin/racingcar/domain/RaceResult.kt

```diff
@@ -1,3 +1,25 @@
 package racingcar.domain
 
-data class RaceResult(val roundHistories: List<RoundHistory>)
+import racingcar.domain.exception.CannotFindWinnerException
+
+data class RaceResult(val roundHistories: List<RoundHistory>) {
+    fun getWinners(): List<String> {
+        val winners = getLastRoundCarInfos()
+            .groupBy { (_, position) -> position }
+            .toSortedMap().values
+            .last()
+
+        return winners.map { (name, _) -> name }
```

Comment:
아래와 같이 람다 프로퍼티가 1개인 경우에는 `it`을 사용하도록 변경해보면 어떨까요? 😃 
```suggestion
            .groupBy { it.second }
            .toSortedMap().values
            .last()

        return winners.map { it.first }
```